docs/getting-started/requirements.md

```
@@ -0,0 +1,128 @@
+### Identifying Your Hardware
+
+```bash
+# Check for PCIe devices
+lspci -nn | grep -i "14c3:7925\|14c3:7921"
+
+# Check for USB devices
+lsusb | grep -i "0e8d:7961"
+```
+
+Example output for MT7925:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)